### ISSUE TITLE
Basic integration test for Sovereign rollups

### ISSUE THREAD

**Issue by soareschen** [informalsystems/hermes-sdk#99]
Part of: #10 
Prerequisites: #94

## Summary

After #94 is implemented, we will write some basic tests to ensure that the test infrastructure for Sovereign rollups is setup properly, and that we are getting proper results from the JSON RPC APIs.

The tests would perform basic operations on a vanilla Sovereign rollup *without* the IBC stack from `sovereign-ibc`. This is so that we can run the basic tests on any Sovereign rollups before they integrate with `sovereign-ibc`.

Eventually in future PRs, the basic Sovereign test suite should test that all non-IBC dependencies for Hermes is implemented by a custom Sovereign rollup. This should make it such that a Sovereign rollup that pass the tests would be able to support IBC by as simple as importing `sovereign-ibc`.

The initial tests would cover basic operations such as:

- Sending transactions
- Local token transfer
- Queries on transactions, events, and height.
- DA-related queries, such as which DA height the rollup is at.

The initial tests would *not* be querying the ZK proof APIs. Those are complicated enough to be tracked by a separate issue.

**Comment by soareschen** [comment]
The basic integration tests have already been implemented. We can close this now and track further progress in separate issues.
